### DISCUSSION TITLE
Zigbee + Bluetooth adapters

### DISCUSSION THREAD

**Discussion by Andrewkha** [Koenkk/zigbee2mqtt#19484] [Devices]
Hi!

I'm very new to the concept but I already have several Bluetooth temperature sensors. As far as I understand to use the Zigbee ecosystem I would need new temperature sensors that support Zigbee protocol. But just to check - probably there are adaptors that support both Zigbee and Bluetooth? Appreciate if you could recommend any or confirm that I would need new temperature sensors...

**Comment by lux73** [comment]
there are mulitprotocol Adapter availalbe

but AFAIK its not possible to use both protocols the same time from the same stick

**Reply by Andrewkha** [reply]
Do you have the examples? I'd explore documentation more in details

**Reply by lux73** [reply]
https://www.nxp.com/products/wireless-connectivity/multiprotocol-mcus/k32w-usb-dongle-for-bluetooth-le-zigbee-and-thread-networks:OM15080-K32W

BUT be aware this Device is not recommended either i don't know if the zigbee Stack installed is z2m compatible ^^
